HabitRPG/HRPGTopHeaderNavigationController.m

```diff
@@ -226,8 +231,17 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
     return YES;
 }
 
-- (void)setState:(HRPGTopHeaderState)state {
-    _state = state;
+- (void)setShouldHideTopHeader:(BOOL)shouldHideTopHeader{
+    if (_shouldHideTopHeader != shouldHideTopHeader) {
+        _shouldHideTopHeader = shouldHideTopHeader;
+        if (shouldHideTopHeader) {
+            [self hideHeader];
+        }else{
```

Comment:
Thank you, good catch! I'm sure there are more mistakes like theses, I'll go back and fix them.